exchanges/okcoin/okcoin.go

```diff
@@ -202,7 +203,7 @@ func (o *Okcoin) GetCandlesticks(ctx context.Context, instrumentID string, inter
 	if err != nil {
 		return nil, err
 	}
-	return ExtractCandlesticks(resp)
+	return ExtractCandlesticks(resp), nil
```

Comment:
In the type CandlestickData there is a field `QuoteTradingVolume` which is being set and field `TradingVolumeInQuote` which is not being set and I think can be removed entirely because they seem like the same thing. Can you confirm?

Reply:
`volCcyQuote	String	Trading volume, the value is the quantity in the quote currency. Not enabled. Please disregard.`
I have removed it.